test/src/pages/institutional-partnership/participants.test.js

```diff
@@ -3,16 +3,14 @@ import shellBus from '~/components/shell/shell-bus';
 import {makeMountRender} from '../../../helpers/jsx-test-utils.jsx';
 
 describe('Participants', () => {
-    it('handles click', () => {
+    it('handles click', (done) => {
         const wrapper = makeMountRender(Participants, {icons: []})();
-        const clickables = wrapper.find('.show-established-partners');
 
-        expect(clickables).toHaveLength(1);
-        const dialogShown = new Promise((resolve) => {
-            shellBus.on('showDialog', resolve, 'once');
-            clickables.at(0).simulate('click');
-        });
-
-        return dialogShown;
+        done();
```

Comment:
I'm confused by why `done` was needed here, I don't see where makeMountRender
 is being waited on.  I also typically just make the it block `async` and wait vs using `done`